[IMP] rental: revised physical products page (#18222)
odoo/documentation: content/applications/sales/rental/configure_products/products.rst

Reviewer: ```suggestion
:menuselection:`Rental app --> Products --> Products`, then click the desired product.
```

Reviewer: ```suggestion
Open the *Prices* tab and click :guilabel:`Add a price`. Select the desired
```

Author: Conferred with Felicia that it's ok to bold tabs if they are part of the instructions.

Reviewer: As an FYI, this design has changed at some point, and if you're updating the other documentation in this file, you should update this as well. There's a pricelist form now, with a table format rather than a pop-up window.

Author: I think this is a case of choosing the wrong word. I didn't mean a pop-up window with ' window displays'. I shall change it to a page or a form. I checked the Pricelist 19.1 docs, and it looks like my demo.

Reviewer: ```suggestion
   Next, go to the :menuselection:`Inventory app --> Configuration --> Locations`. Create a new location
```

========================

The Odoo **Rental** app allows users to customize scheduling, pricing, and inventory for physical
rental products that require stock movement, otherwise known as *Goods*. Users can set up multiple
pickup and drop-off locations and track rental products by serial number.

Settings
========

The **Rental** app offers many app-integration features. Depending on the installed Odoo apps,
specific settings are available. To learn more about the default settings for rental products, refer
to the :ref:`rental/product_types/configuration` section on the *Rental product types* page.

To access the **Rental** app's settings, navigate to :menuselection:`Rental app --> Configuration
--> Settings`. If only the **Rental** app is installed, then the :guilabel:`Configuration` menu is
disabled and can't be accessed.

The following configurations assume the **Rental**, **Inventory**, and **Sales** apps are installed.

.. _rental/products/physical-products:

Create a new physical product
=============================

> To set up a new physical rental product, go to the :menuselection:`Rental app --> Products`, then

Reviewer: ```suggestion
To set up a new physical rental product, go to the :menuselection:`Rental app --> Products --> Products`, then
```

click :guilabel:`New`. On the rental product form, fill out each tab accordingly:

General Information tab
-----------------------

In the new product window, the :guilabel:`Sales` checkbox is already ticked by default. Select the

> :guilabel:`Product Type` as :guilabel:`Goods`.

Reviewer: ```suggestion
In the new product window, the :guilabel:`Sales` checkbox is already selected by default. Select :guilabel:`Goods` as the :guilabel:`Product Type`.
```

1. We are moving away from "ticked" and moving to "select" or "clear." It generally translates better.
2. I made a suggestion for the second sentence because it reads slightly awkward. Alternate suggestion: "In the Product Type menu, select Goods."

for the selected duration.

If the *eCommerce* module is installed, the :guilabel:`Pickup` and :guilabel:`Return` fields are
displayed for every :guilabel:`Periodicity` field option except :guilabel:`Hours`. The
:guilabel:`Pickup` and :guilabel:`Return` times only apply to online rental orders.

All additional rental rates can be configured on the :ref:`rental/products/prices-tab`, but they are
restricted to the configured :guilabel:`Periodicity` value selected in the *Sales* tab. In other
words, a rental product can only have one :guilabel:`Periodicity` value configured at a time.

.. image:: products/rental-periodicity.png
   :alt: Sample of the Rental product's Periodicity and Padding time in the Rental app.

Attributes & Variants tab
-------------------------

.. important::
   The :guilabel:`Variant` feature in the **Inventory** app must be enabled for this tab to display.

Add the appropriate :ref:`attribute and its values <products/variants/attributes>` by clicking the
:guilabel:`Add a line`. Attributes and values are useful for keeping the product library manageable,
tracking and differentiating the inventory, and providing more detailed reports. Examples of rental

> variants for a *Good* product are: sizes, brand, color, and material.

Reviewer 2: <img width="934" height="343" alt="Image" src="https://github.com/user-attachments/assets/05ea3c22-0748-4a3f-8a55-8832131f9092" />

```suggestion
variants for a *Goods* product are: sizes, brand, color, and material.
```

since the UI term is Goods or Service

.. _rental/products/prices-tab:

Prices tab
----------

.. important::

>    The **Sales** app must be installed, the :guilabel:`Pricelist` feature must be enabled for this

Reviewer 2: ```suggestion
   The **Sales** app must be installed, the *Pricelist* feature must be enabled for this
```

since the UI element isn't on the screen rn, let's do italics

For the :guilabel:`Category` field, select :guilabel:`Goods` from the drop-down menu or create a new

> category by typing in the name and clicking :guilabel:`Create`.

Reviewer: I am admittedly slightly confused by these paragraphs. I will make a few suggestions here:

1. We want to be sure to use "select" instead of "tick."
2. Are these different options that we can select for? I guess what I'm asking: Is this "By Quantity" vs "By Unique Serial Number"? If so, we should probably present them as such. It could be as simple as, "To track by quantity..." and "To track by unique serial number..."
3. Looking in runbot, some of the sample rental products are set to the Rental category. Is "Goods" required? The language used indicates that it might be required.
4. We again have the "Select the Product Type as Goods" language. I think my prior suggestions work better here.

Author: That was a large typo on my part. The duplicated info about Unique serial numbers should've been deleted.

Author: We recommend setting a category for the product, it can be Goods or a custom one.

.. image:: products/new-product.png
   :alt: The new product view in the Rental app.

.. _rental/products/base-rental-period-price:

Set a base rental period and price
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Set up a base rental rate by entering the lowest rental price in the :guilabel:`Sales Price` field
of the *General Information* tab. Next, click the :guilabel:`Sales` tab, then in the *Rental*
section select a unit of time from the :guilabel:`Periodicity` drop-down menu.

Then enter the :guilabel:`Padding Time` to make the product unavailable for pickup for the
configured duration.

> All additional rental rates can be configured on the :ref:`rental/products/prices-tab`, but they are

Reviewer: ```suggestion
All additional rental rates can be configured in a pricelist on the :ref:`rental/products/prices-tab`, but they are
```

Author: Technically, any rental rate added to the rental product's Prices tab is a pricelist rule. Not a new pricelist. I will clarify that in the Prices tab section and leave the sentence as is.

restricted to the configured :guilabel:`Periodcity` value selected in the *Sales* tab. In other
words, a rental product form can only have one :guilabel:`Periodocity` value configured at a time.

.. image:: products/rental-periodicity.png
   :alt: Sample of the Rental product's Periodicity and Padding time in the Rental app.

Attributes & Variants tab
-------------------------

.. important::
   The :guilabel:`Variant` feature in the **Inventory** app must be enabled for this tab to display.

Click :guilabel:`Add a line`, then select a default option from the :guilabel:`Attribute` drop-down
menu. To create a new one, enter the name, then click :guilabel:`Create and edit…` to configure the

> :ref:`attribute and its values <products/variants/attributes>`.

Reviewer: What about the values? We might need to specify that they should be selected here, as well.

Also, what is the "default option"? Are we saying at least one variant needs to be specified? Or can the word "default" be safely omitted from the sentence?

Author: I revised this section to add value and make the hyperlink more obvious. I am trying to cut down on redundant instructions.

.. _rental/products/prices-tab:

Prices tab
----------

.. important::
   The **Sales** app must be installed and the :guilabel:`Pricelist` feature enabled for this tab to
   display.

There are two ways to configure additional rental rates in the **Rental** app: :ref:`Pricelists
method <rental/products/pricelist>` and the :ref:`Prices tab method <rental/products/price-tab>`. To

> learn how The **Rental** app follows specific conditions when using pricelists, refer to the

Reviewer: ```suggestion
learn how the **Rental** app follows specific conditions when using pricelists, refer to the
```

> :ref:`rental/rental-pricelist-rules` section on the *Rental* page.

Reviewer: ```suggestion
:ref:`rental/rental-pricelist-rules` section of the *Rental* documentation.
```

~~~~~~~~~~~~~~~~~~~~~~~~~~~

Rental rates can also be configured as a new price rule to an existing pricelist using the
:guilabel:`Prices` tab on the product form. Navigate to :menuselection:`Rental app --> Products -->
Products`, then click the desired product.

Click the :guilabel:`Prices` tab and click :guilabel:`Add a price`. Select the desired
:guilabel:`Pricelist`, then enter the minimum time required for the price change to trigger in the
:guilabel:`Min. Quantity` column. The :guilabel:`Min. Quantity` column is based on the configured
:guilabel:`Periodicity` value chosen in the :guilabel:`Sales` tab.

Lastly, enter the :guilabel:`Price` rate. Click the :icon:`fa-cloud-upload` :guilabel:`(Save
manually)` icon near the top to save.

.. example::
   A bike rental business rents out its bikes on an hourly basis but offers a 20% discount for
   summer break. The regular hourly rate for their bikes is $20.

   Enter the :guilabel:`Sales Price` in the *General Information* tab of the product form, then
   click the :guilabel:`Sales` tab to configure the :guilabel:`Periodicity` and :guilabel:`Padding
   Time`.

   .. image:: products/rental-sales-tab-rental-section.png

>       :alt: Add alt text.

Reviewer 2: Remember to define the alt text for this image!

   Using the Pricelist method, navigate to :menuselection:`Rental app --> Products --> Pricelists`
   and click :guilabel:`New`. Configure :guilabel:`Pricelist Rules` for the 20% discount.

   .. image:: products/example-pricelist-method.png
       :alt: Sample of a rental product with the custom rental pricelist applied.

   Using the :guilabel:`Prices` tab method, navigate to :menuselection:`Rental app --> Products -->
   Products` and click the bike product. Click the :guilabel:`Prices` tab, then add a new discounted
   price for the hourly rate. To add the :guilabel:`Validity` column, click the

>    :icon:`oi-settings-adjust` :guilabel:`Optional dropdown` icon and select :guilabel:`Validity`.

Reviewer 2: For icon formatting, we use the hover text and put it in parantheses. Can you double-check that when you hover over this icon in the runbot, it says "Optional dropdown" and then make sure to put parantheses around it?

checkbox.

.. image:: products/rental-transfers-checkbox.png
   :alt: Sample of the Rental settings with the Rental Transfers enabled.

.. note::
   The **Inventory** app automatically creates an internal default location once the
   :guilabel:`Rental Transfers` feature is enabled. Odoo uses the new default location,
   :guilabel:`Customer/Rental`, to track products during the rental period (moving them from
   :guilabel:`Stock` to :guilabel:`Customer/Rental` upon rental, and back upon return). Do not
   modify to avoid corrupting inventory tracking.

.. _rental/configure_products/multi-location:

Multi-location management and transfers
=======================================

.. important::
   The **Inventory** app must be installed to set up this configuration. The **Inventory** app
   automatically creates an internal default location once the :guilabel:`Rental Transfers` feature
   is enabled. Odoo uses the new default location, :guilabel:`Customer/Rental`, to track products
   during the rental period (moving them from :guilabel:`Stock` to :guilabel:`Customer/Rental` upon
   rental, and back upon return).

>    Do not modify to avoid corrupting inventory tracking.

Reviewer 2: We say "Do not modify to avoid corrupting inventory tracking" in line 217 and 202. Was this intentional?

Also, what are we not modifying? the Customer/Rental location? It might be clearer to say "Do not modify Customer/Rental to avoid corrupting inventory tracking"

Author: Yes, it's intentional. I kept it in both sections in case the reader jumped to a specific section.

>        :alt: Sample of a rental product with the custom rental pricelist applied.

Reviewer: Again, this is not what I see in runbot and therefore would recommend updating if it's indeed changed, including the screenshot.

   Using the :guilabel:`Prices` tab method, navigate to :menuselection:`Rental app --> Products -->
   Products` and click the Bike product. Click the :guilabel:`Prices` tab, then add a new price for
   the daily rate.

   .. image:: products/example-rental-prices-tab.png
      :alt: Sample of a rental product's Price tab.

Configure a physical rental product for product tracking
========================================================

.. important::
   To configure a physical rental product for product tracking, the **Inventory** app must be
   installed, and :guilabel:`Lots & Serial Numbers` must be enabled.

Go to the :menuselection:`Rental app --> Products`, then click :guilabel:`New`. In the new product
window, the :guilabel:`Sales` checkbox is already ticked by default.

Select the :guilabel:`Product Type` as :guilabel:`Goods`. Tick the :guilabel:`Track Inventory`
checkbox and select either :guilabel:`By Lots` or :guilabel:`By Unique Serial Number` from the

> drop-down menu.

Reviewer: ```suggestion
Select :guilabel:`Goods` as the :guilabel:`Product Type`. Select the :guilabel:`Track Inventory`
checkbox and select either :guilabel:`By Lots` or :guilabel:`By Unique Serial Number` from the
```

For the :guilabel:`Category` field, select :guilabel:`Goods` from the drop-down menu or create a new
category by typing in the name and clicking :guilabel:`Create`. Configure :ref:`basic rental rate
<rental/products/base-rental-period-price>` and any :ref:`additional rates

> <rental/products/prices-tab>`. Click the (Save manually) icon near the top to save.

Reviewer: You're missing the fa-cloud-upload icon here.

Rental Transfers feature
========================

The :guilabel:`Rental Transfers` feature automatically creates a delivery receipt when the rental
product is picked up and a return receipt when it is returned to stock. Documenting stock movement
creates a clean paper trail and has a variety of uses:

- Tracking high-value products.
- Tracking stock levels across multiple stores or warehouse locations.

> - Tracking products between different store locations that allow pick up and returns.

Reviewer: I love bringing the value-add to this, FYI!

Odoo downloads a PDF detailing all information about the current status of the rented items.

.. seealso::
   - :doc:`../../../inventory_and_mrp/inventory`

>    - `Tutorial: Configuring a rental product <https://youtu.be/CE-SahTUC9A?si=APacZmYDIsVnHOnj>`_

Reviewer 2: When listing tutorials, the convention is "Odoo Tutorials: <title>"

```suggestion
   - `Odoo Tutorials: Configuring a rental product <https://youtu.be/CE-SahTUC9A?si=APacZmYDIsVnHOnj>`_
```
